D.py

```diff
@@ -0,0 +1,2 @@
+def godmorning():
```

Comment:
```suggestion
def godmorning():
```
godmorningではなくgoodmorningではないでしょうか！